force-app/main/default/reports/NPSP_Outbound_Fund_Reports/Unscheduled_Disbursements.report-meta.xml

```diff
@@ -26,7 +26,7 @@
         <aggregateTypes>Sum</aggregateTypes>
         <field>Funding_Request__c.Total_Cancelled__c</field>
     </columns>
-    <description>List of Disbursements associated with an awarded Funding Request that need to be scheduled.</description>
+    <description>List of awarded Funding Requests that have available funds to be scheduled for disbursement.</description>
```

Comment:
@nicolersmith  - Hi Nicole, this description needs to be updated. Annie suggested the description below. Please let us know if wee need to change it.

Reply:
This looks good to me. Thanks!